Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -131,13 +139,13 @@ private void ValidationErrorHandler(object sender, EventArgs e)
         private void NextButton_Click(object sender, RoutedEventArgs e)
         {
             bool bErrors = HasValidationsIssues();
-            UpdateFinishButton();
             if (bErrors)
             {                
                 return;
-            }            
+            }
 
             mWizard.Next();
+            UpdateFinishButton();
             UpdatePrevNextButton();
```

Comment:
please check the impact on Agent/Environments/API wizards to make sure all next/finish buttons there still works as expected after this change

Reply:
Tested